zipkin-server/README.md

```diff
@@ -268,6 +268,16 @@ The following apply when `STORAGE_TYPE` is set to `elasticsearch`:
                            to 0 as it would mean a machine failure results in data loss.
     * `ES_USERNAME` and `ES_PASSWORD`: Elasticsearch basic authentication, which defaults to empty string.
                                        Use when X-Pack security (formerly Shield) is in place.
+    * `ES_CREDENTIALS_FILE_PATH`: Elasticsearch credentails properties file path, which defaults to empty
```

Comment:
ES_CREDENTIALS_FILE_PATH is chatty, please use ES_CREDENTIALS_FILE

Reply:
done